src/pages/upload/index.tsx

```diff
@@ -47,6 +51,7 @@ export const getServerSideProps: GetServerSideProps = async function ({req}) {
             'image': image.url,
           },
         eggheadInstructorId,
+        _id
```

Comment:
Adding `_id` to this GROQ query because that is needed for the collaborator `reference` on the `course`.